src/i18n/messages/en.messages.json

```diff
@@ -38,10 +38,10 @@
       "message": "Those who searched for <span class=\"x-text x-text--bold x-font-size--05\">\"{query}\"</span> also viewed:"
     },
     "nextQueryPreview": {
-      "title": "Next Queries",
+      "message": "Those who searched for <span class=\"x-text x-text--bold x-font-size--05\">\"{query}\"</span> also viewed:",
```

Comment:
There are no `"` in the designs for this message's query.

Reply:
Mobile messages are different for this property, but I think there is a typo. Can we confirm that we want to leave them like this?

Reply:
Yes, if you check the task, it said that both messages have to be the same